weather_app/weather/models.py

```diff
@@ -10,3 +13,110 @@ def __str__(self):
     class Meta:
         verbose_name_plural = 'cities'
 
+
+class Jugador(models.Model):
+    nombre = models.CharField(max_length=20)
+    apellido = models.CharField(max_length=20)
+
+class Deporte(models.Model):
+    jugador = models.ForeignKey(Jugador, on_delete=models.PROTECT)
+    deporte_practica = models.CharField(max_length=100)
+
+
+class Cantante(models.Model):
+    nombre = models.CharField(max_length=25)
+    apellido = models.CharField(max_length=25)
+
+class Genero(models.Model):
+    cantante = models.CharField(Cantante, on_delete=models.CASCADE)
```

Comment:
```suggestion
    cantante = models.ForeignKey(Cantante, on_delete=models.CASCADE)
```

Recuerda que las foreign key van con el tipo de campo `models.ForeignKey`

Reply:
ya quedo arreglado